pkg/cloud/azurestack/assets/cloud-node-manager-daemonset.yaml

```
@@ -44,6 +44,32 @@ spec:
         - effect: NoSchedule
           key: node.kubernetes.io/not-ready
           operator: Exists
+      initContainers:
+        - name: azure-inject-credentials
+          image: quay.io/openshift/origin-cluster-cloud-controller-manager-operator
+          command:
+            - /azure-config-credentials-injector
+            - --cloud-config-file-path
+            - /tmp/cloud-config/cloud.conf
+            - --output-file-path
+            - /tmp/merged-cloud-config/cloud.conf
```

Technically this should be:
```suggestion
          command:
            - /azure-config-credentials-injector
          args:
            - --cloud-config-file-path=/tmp/cloud-config/cloud.conf
            - --output-file-path=/tmp/merged-cloud-config/cloud.conf
```
I'm not a fan of splitting the flag name and value over separate lines so would prefer to use an equals and join them